src/server/saved_objects/service/lib/repository.js

```diff
@@ -50,6 +51,7 @@ export class SavedObjectsRepository {
     this._index = index;
     this._mappings = mappings;
     this._schema = schema;
+    this._includeHiddenTypes = includeHiddenTypes || [];
```

Comment:
Can we put this default in the destructuring assignment like `onBeforeWrite`?

Reply:
Yeah makes more sense that way.